tests/sles4sap/publiccloud/qesap_ansible.pm

```diff
@@ -67,16 +109,39 @@ sub run {
         $run_args->{site_b} = $instance if ($instance_id ne $master_node);
     }
 
-    get_var('QESAP_DEPLOYMENT_IMPORT') ?
-      record_info('IMPORT OK', 'Importing infrastructure successfully.') :
-      record_info('DEPLOY OK', 'Ansible deployment process finished successfully.');
+    get_var('QESAP_DEPLOYMENT_IMPORT')
+      ? record_info('IMPORT OK', 'Importing infrastructure successfully.')
+      : record_info('DEPLOY OK', 'Ansible deployment process finished successfully.');
 
     return unless $ha_enabled;
 
-    record_info('Instances:', "Detected HANA instances:
+    record_info(
+        'Instances:', "Detected HANA instances:
     Site A (PRIMARY): $run_args->{site_a}{instance_id}
-    Site B: $run_args->{site_b}{instance_id}");
+    Site B: $run_args->{site_b}{instance_id}"
+    );
     return 1;
 }
 
+sub clean_up {
```

Comment:
is it a "clean_up only for the timeout" method?

Reply:
Currently yes.

Reply:
please mind to rename the function to `cleanup_for_retry` to avoid someone in the future to use it in other context.

Reply:
Done.